python-sdk/tests/benchmark/config.json

```diff
@@ -34,14 +19,6 @@
       "rows": 160,
       "size": "10 KB"
     },
-    {
-      "conn_id": "bigquery",
-      "file_type": "csv",
-      "name": "hundred_kb",
-      "path": "gs://astro-sdk/benchmark/trimmed/tate_britain/artist_data_100kb.csv",
-      "rows": 748,
-      "size": "100 KB"
-    },
```

Comment:
Why these removals?

Reply:
These were temporary attempts to see the test running on BQ while I was troubleshooting with @utkarsharma2